### PR TITLE
refresh op shelf detail page (bug 1103300)

### PR DESCRIPTION
**Pull request by ngokevin**
- re-use collection stuff. pretty straightforward.
- will need to do stuff with op shelf colors since those are hard-coded via CSS. but will handle at the curves stage.
- minus 111 LOC. bam

screen shot 2015-01-13 at 1 15 05 pm
screen shot 2015-01-13 at 1 43 14 pm

### CODE REVIEW

File: src/media/js/views/feed/feed_shelf.js [hunk 1]
```diff
@@ -10,14 +11,11 @@ define('views/feed/feed_shelf', ['jquery', 'l10n', 'utils', 'utils_local', 'z'],
 
         var slug = args[0];
         builder.start('feed/feed_shelf.html', {
-            landing: true,
             slug: slug
```
Comment (spasovski) [inline src/media/js/views/feed/feed_shelf.js]: Can probably have `builder.start()` be all one-line now.

File: src/templates/_macros/feed_item.html [hunk 2]
```diff
@@ -188,19 +188,25 @@ <h1 class="name">{{ coll.name|translate(coll)|safe }}</h1>
 {% endmacro %}
 
 {% macro collection_landing(coll) %}
+  {# Used both for collections and shelves. Reduce, reuse, recycle. #}
```
Comment (spasovski) [inline src/templates/_macros/feed_item.html]: Probably fine until we add classes for the colors.

File: src/media/js/views/feed/feed_shelf.js (hunk 1 above)
Comment (ngokevin) [inline src/media/js/views/feed/feed_shelf.js]: could, but most builder.starts look like this so it's easy to spot. plus we always might add more vars

**Comment by spasovski** [comment]
You'll want to add the 'app' class to the direct-children li's of `.app-list` going forward. My PR makes this change.

**Comment by spasovski** [comment]
r+wc seems good locally

**Comment by ngokevin** [comment]
<mkt-app-list> sounding nice now

**Comment by ngokevin** [comment]
```app``` is way too general. you'll want to do ```app-list-app``` instead. i'll merge this, and you can add it here once you change it in your PR.

**Comment by ngokevin** [comment]
thanks

**Pull request merged**